rest-helix/src/main/java/com/github/twitch4j/helix/TwitchHelix.java

```diff
@@ -728,6 +729,114 @@ HystrixCommand<UpdatedDropEntitlementsList> updateDropsEntitlements(
         UpdateDropEntitlementInput input
     );
 
+    /**
+     * Creates a new EventSub conduit.
+     *
+     * @param authToken  App access token.
+     * @param shardCount The number of shards to create for this conduit.
+     * @return ConduitList
+     */
+    @RequestLine("POST /eventsub/conduits")
+    @Headers({
+        "Authorization: Bearer {token}",
+        "Content-Type: application/json"
+    })
+    @Body("%7B\"shard_count\":{shard_count}%7D")
+    HystrixCommand<ConduitList> createConduit(
+        @Param("token") String authToken,
+        @Param("shard_count") int shardCount
```

Comment:
```suggestion
        @NotNull @Param("shard_count") int shardCount
```

Reply:
agreed on all, but unnecessary for primitives